cluster/network.go

```diff
@@ -0,0 +1,23 @@
+package cluster
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/rancher/rke/k8s"
+	"github.com/rancher/rke/pki"
+)
+
+func (c *Cluster) DeployNetworkPlugin() error {
+	logrus.Debugf("[Network] Setting network plugin: %s", c.NetworkPlugin)
+	deployerHost := c.ControlPlaneHosts[0]
+	kubectlCmd := []string{"apply -f /network/" + c.NetworkPlugin + ".yaml"}
```

Comment:
Looks very hacky to me, the way we build the commands...Consider for example that there can be multiple files in the future, multiple services (dns). Lets think of more generic way of deploying all the addons. May be similar to kubernetes-package?

Reply:
The intention here is to create a general use kubectl container. It can be used to deploy any manifest on we add later or to run ad-hoc commands.